.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -24,7 +24,7 @@ Ensure that you use markdown syntax for articles, we do not accept `HTML` or `CS
 
 **Commits and Squashing:**
 - Your Pull Request/Submission should contain one single commit.
-- For multiple commits follow the [guide to squash them](git-rebase#squashing-multiple-commits-into-one).
+- For multiple commits follow the [guide to squash them](Git-Squash).
```

Comment:
@Rafase282 oops! I remember removing the comment right after making the comment. But seems like it didn't remove. Sorry!

The relative will work. :+1: